src/test/java/seedu/duke/flashutils/commands/FlashbangCommandTest.java

```diff
@@ -1,30 +1,27 @@
 package seedu.duke.flashutils.commands;
 
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import seedu.duke.flashutils.types.Card;
 import seedu.duke.flashutils.types.FlashCardSet;
 import seedu.duke.flashutils.utils.StubStorage;
```

Comment:
I think both Card and StubStorage are unused imports causing the checkstyle to detect it as an error